src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -101,7 +101,7 @@ public void createGame(@RequestHeader(value = "authorization", required = false)
   }
 
     @GetMapping("users/{userId}/games/blackCards")
-    @ResponseStatus(HttpStatus.CREATED)
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Is this in line with the api excel?

Reply:
a get should never create anything.. and yes the excel states code 200